core/lib/dal/migrations/20250213120658_create_etherscan_verification_requests_table.up.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE IF NOT EXISTS etherscan_verification_requests (
+    contract_verification_request_id BIGINT PRIMARY KEY,
+    status CHARACTER VARYING(32) NOT NULL,
+    processing_started_at TIMESTAMP,
+    attempts INT NOT NULL DEFAULT 0,
+    etherscan_verification_id CHARACTER VARYING(128),
+    result TEXT,
+    error TEXT,
+    created_at TIMESTAMP NOT NULL,
+    updated_at TIMESTAMP NOT NULL,
+    CONSTRAINT contract_verification_requests_id_fkey FOREIGN KEY (contract_verification_request_id) REFERENCES public.contract_verification_requests (id) ON UPDATE NO ACTION ON DELETE NO ACTION
+);
+
+CREATE INDEX IF NOT EXISTS contract_verification_requests_status_processing_at_idx
+ON contract_verification_requests (status, processing_started_at) 
+WHERE status = 'queued' OR status = 'in_progress';
```

Comment:
This new index has nothing to do with Etherscan verification, but I noticed that the current index is not being used, so I’m adding a new one. To be on the safe side, I decided not to remove the old one.